crates/trigger/src/cli/variables.rs

```diff
@@ -0,0 +1,44 @@
+use spin_core::async_trait;
+use spin_factors::RuntimeFactors;
+use spin_factors_executor::ExecutorHooks;
+use spin_variables::{VariableProviderConfiguration, VariableSourcer};
+
+/// Implements TriggerHooks, sorting required variables
+pub struct VariableSorterExecutorHooks {
```

Comment:
Not sure about the naming here.  Why is it an "Executor" hook?

Reply:
The naming choice was to reflect its implementation of `the ExecutorHooks` trait